examples/binding_affinity_prediction/README.md

```diff
@@ -40,6 +88,15 @@ PDBBind_refined_pocket_stratified, PDBBind_refined_pocket_temporal}, dataset and
 | Refined | Stratified       | 1.6376   | 0.3050  |
 | Refined | Temporal         | 1.2457   | 0.3438  |
 
+#### PotentialNet
+
+| Training Set | Test Set| Splitting Method | PDBBind version | Train R2 (std) | Valication R2 (std) | Test R2 (std) |
+| ------------ | --------| ---------------- | --------------- | -------------- | ------------------- | ------------- |
+| Refined      | Core    | Random           | v2007           | 0.9305 (0.0106)| 0.4161 (0.0217)     |0.8980 (0.0102)|
+| Refined      | Core    | Random           | v2015           | 0.8934 (0.0112)| 0.4227 (0.0170)     |0.8117 (0.0135)|
+
+The results are computed over 100 trials and R2 from the best epoch is reported. Note that core set is not removed from the refined set.
```

Comment:
1. By "from the best epoch", do you mean the epoch with the best validation performance?
2. "Note that core set is not removed from the refined set." If this is the practice from the paper, you should mention it.

Reply:
1. Was best test epoch, now changed to best val epoch.
2. It is unclear whether they removed core from refined for training. So I will experiment and report performance on both cases.